asciidoc/volume3/biceps-extension-provisions/tf3-ch-8.3.2.9-extension-equipment-identifier.adoc

```diff
@@ -0,0 +1,54 @@
+[#vol3_clause_equipment_identifier]
+====== Equipment Identifier
+
+BICEPS does not provide means to convey any metadata that allows for identification of <<acronym_mds>>s or <<acronym_vmd>>s inside an <<acronym_mdib>> other than by UDIs.
+UDIs however are not unique when used in different jurisdictions or for multiple instances of a <<acronym_samd>>.
+
+This extension fulfills the need to identify <<acronym_mds>>s and <<acronym_vmd>>s across re-initializations of a <<actor_somds_provider>> by means of stable and globally unique URIs that are constant across re-initializations of the <<actor_somds_provider>>.
+
+[sdpi_level=+1]
+====== Model
+
+The Equipment Identifier XML Schema is available in <<vol3_appendix_a_xml_schemas_equipment_identifier>>. <<vol3_example_extension_equipment_identifier>> shows an exemplary XML instance of a <<vol2_clause_dev_30_message_getmdibresponse, {var_label_dev_30_message_getmdibresponse}>> including the Equipment Identifier extension in the descriptive part of the MDIB.
+
+.<<acronym_mds>> and <<acronym_vmd>> with Equipment Identifier extension
+[#vol3_example_extension_equipment_identifier]
+====
+[source,xml]
+----
+include::../../listings/vol3-clause-biceps-content-example-equipment-identifier.xml[]
+----
+====
+
+[sdpi_level=+1]
+====== Requirements
+
+.R0014
+[sdpi_requirement#r0014,sdpi_req_level=shall]
+****
+For the `sdpi:EquipmentIdentifier` extension, a <<actor_somds_provider>> shall use a stable, globally unique URI that is constant across re-initializations of the <<actor_somds_provider>> and uniquely refers to a physical or virtual entity.
+****
+
+.R0015
+[sdpi_requirement#r0015,sdpi_req_level=should]
+****
+For the `sdpi:EquipmentIdentifier` extension, a <<actor_somds_provider>> should use URI-encoded UUIDs or OIDs.
+****
+
+.R0016
+[sdpi_requirement#r0016,sdpi_req_level=shall]
+****
+To test if two equipment identifiers are equal, a <<actor_somds_participant>> shall perform case-sensitive string comparison.
+****
+
+.R0017
+[sdpi_requirement#r0017,sdpi_req_level=shall]
+****
+A <<actor_somds_provider>> shall add the `sdpi:EquipmentIdentifier` extension to each pm:MdsDescriptor in its <<acronym_mdib>>.
+****
+
+.R0018
+[sdpi_requirement#r0018,sdpi_req_level=should]
+****
+A <<actor_somds_provider>> should add the `sdpi:EquipmentIdentifier` extension to each pm:VmdDescriptor in its <<acronym_mdib>>.
```

Comment:
Analog to my comment on R0017, this is quite strict for a VMD. "May" might be useful or a condition, e.g., to removable subsystems represented by an VMD?

Reply:
I'm going to add a condition as you proposed.